Cleanup and streamline

garrettwong opened the pull request:
- Remove dependency on config.json
- Remove bloated files (e.g. cloudbuild.yaml)
- Remove config dependencies from files
- Update `source.env`config file with additional parameters to use the default of 0.0.0.0 and Port 8080 on the API Layer and include the PROJECT_ID as it is used in the Infra Deployment Examples.  
- Update README to reflect removal of the config.json step and reflect updated source.env steps.

joecheuk commented on README.md:
```diff
@@ -49,34 +49,17 @@ Navigate to forseti-api/.  Create a `source.env` file, which will should be sour
-  "oauth2Callback": "http://localhost:8080/auth/google/callback",
-}
+export FORSETI_SERVER_VM_CHANNEL="0.0.0.0:50051"
+export FORSETI_DATA_MODEL_HANDLE="21254f1de747879237a95cb552e80844"
```
Do we expect people to update these variables ?

joecheuk commented on forseti-api/README.md:
```diff
@@ -1,47 +1,37 @@
-  "oauth2Callback": "http://localhost:8080/auth/google/callback",
-}
+export FORSETI_SERVER_VM_CHANNEL="0.0.0.0:50051"
+export FORSETI_DATA_MODEL_HANDLE="21254f1de747879237a95cb552e80844"
```
same here, is there a better placeholder to put here instead of putting a model id?

joecheuk commented on forseti-api/README.md:
```diff
@@ -52,32 +42,35 @@ npm start
-FORSETI_SERVER_VM_CHANNEL=[VM_IP]:[GRPC_PORT]
-FORSETI_DATA_MODEL_HANDLE=[DATA_MODEL_HANDLE_HASH:21254f1de747879237a95cb552e80844]
+FORSETI_SERVER_VM_CHANNEL=0.0.0.0:50051
+FORSETI_DATA_MODEL_HANDLE=21254f1de747879237a95cb552e80844
```
same here.

joecheuk commented on infrastructure/source2.env:
```diff
@@ -0,0 +1,9 @@
+export API_HOST="0.0.0.0"
+export API_PORT="8080"
+export CLOUDSQL_HOSTNAME="104.197.52.78"
```
I think we shouldn't commit these information

garrettwong committed:
Remove bloated files (e.g. cloudbuild.yaml)
Remove config dependencies from files
Update source.envconfig file with additional parameters to use the default of 0.0.0.0 and Port 8080 on the API Layer and include the PROJECT_ID as it is used in the Infra Deployment Examples.
Update README to reflect removal of the config.json step and reflect updated source.env steps.

garrettwong committed:
Merge changes from master

garrettwong committed:
update package.json/lock files

garrettwong committed:
security-fix
